src/main/rx/flysky.c

```diff
@@ -415,6 +422,9 @@ rx_spi_received_e flySkyDataReceived (uint8_t *payload)
 {
     rx_spi_received_e result = RX_SPI_RECEIVED_NONE;
     uint32_t timeStamp;
+#ifdef USE_RX_FLYSKY_SPI_LED
+    static uint16_t led_rxlosscount = 0;
```

Comment:
Local `static` variable definitions should be at the very  beginning of the function. Also, use camel case,
 and no need to make this specific to LED (`rxLossCount`).